ivy/container/manipulation.py

```diff
@@ -1707,6 +1747,47 @@ def zero_pad(
         ivy.Container instance method variant of ivy.zero_pad. This method simply wraps
         the function, and so the docstring for ivy.zero_pad also applies to this method
         with minimal changes.
+
+        Parameters
+        ----------
+        self
+            Input array to pad.
+        pad_width
+            Number of values padded to the edges of each axis. Specified as
+            ((before_1, after_1), … (before_N, after_N)),
+            where N is number of axes of x.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is ``None``.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is ``True``.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is ``False``.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples).
+            Default is ``False``.
+        out
+            optional output array, for writing the result to.
+            It must have a shape that the inputs broadcast to.
+
+        Returns
+        -------
+        ret
+            Padded array of rank equal to x with shape increased according to pad_width.
+
+        Examples
+        --------
+        With :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a = ivy.array([1., 2., 3.]), b = ivy.array([3., 4., 5.]))
+        >>> y = ivy.zero_pad(x, pad_width = [[2, 3]])
```

Comment:
This should be written like the example in array instance method, i.e x.zero_pad(pad_width=....)